<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>-5</y>
      <width>661</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

background-color:#e6f2fe;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QDial" name="hour_dial">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>211</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QDial" name="minute_dial">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>211</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="setalarm_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hour_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="colon_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="minute_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>41</y>
      <width>58</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Am</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pm</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
